gwpy/table/tests/test_table.py

```diff
@@ -119,6 +119,13 @@ def create(cls, n, names, dtypes=None):
     def table(cls):
         return cls.create(100, ['time', 'snr', 'frequency'])
 
+    @classmethod
+    @pytest.fixture()
+    def clustertable(cls):
+        return cls.TABLE(data=[[11, 1, 1, 10, 1, 1, 9],
+                         [0.0, 1.9, 1.95, 2.0, 2.05, 2.1, 4.0]],
```

Comment:
Can you please line up the indentation here:
```suggestion
                               [0.0, 1.9, 1.95, 2.0, 2.05, 2.1, 4.0]],
```

or re-indent the whole thing:

```python
       return cls.TABLE(
            data=[
                [11, 1, 1, 10, 1, 1, 9],
                [0.0, 1.9, 1.95, 2.0, 2.05, 2.1, 4.0],
            ],
            names=['amplitude', 'time'],
        )

Reply:
Sure, pushed